tox.ini

```diff
@@ -89,3 +89,13 @@ commands =
         --confcutdir=tests/integration \
         {posargs:tests/integration}
 passenv = KRB5CCNAME REQUESTS_CA_BUNDLE KRB5_CLIENT_KTNAME CACHITO_TEST_CERT CACHITO_TEST_KEY JOB_NAME
+
+[pytest]
+addopts = --tb=native --verbose --capture=no
```

Comment:
Why --tb=native? It looks less readable than pytest's default output to me

--verbose is already set in `[testenv:integration]` as -vvvv. I think we can keep it only for int. tests, IMO we don't need it for unit tests

What does --capture=no do?

Reply:
ou sry I haven't replied to your comment to pytest section in tox.ini. Sure I'll remove --tb=native and verbose. `--capture=no` allows you to use print in output. Since we have logging setup we don't need that so I can remove that too.

Reply:
actually without capture option our unit tests will fail https://github.com/containerbuildsystem/cachi2/actions/runs/3695607265/jobs/6258245391

Reply:
Huh, we never used the option before :thinking: 

And that's a lot of noise in the unit test output, but let's fix that later